app/actors/hyrax/actors/file_set_actor.rb

```diff
@@ -159,8 +162,37 @@ def unlink_from_work
           work.thumbnail = nil if work.thumbnail_id == file_set.id
           work.representative = nil if work.representative_id == file_set.id
           work.rendering_ids -= [file_set.id]
-          work.save!
+          save(work, use_valkyrie: false)
         end
+
+        def save(af_object, use_valkyrie: false)
+          af_object.save unless use_valkyrie
+
+          adapter = Hyrax.config.valkyrie_metadata_adapter
+          resource = adapter.persister.save(resource: af_object.valkyrie_resource)
+          Wings::ActiveFedoraConverter.new(resource: resource).convert
```

Comment:
Is the conversion back to an af_object necessary?  It isn't assigned to anything and the method returns true.

Reply:
oh shoot, forgot about this one. @revgum asked me to put this in.